src/main/java/seedu/address/model/category/UniqueCategoryList.java

```diff
@@ -43,12 +43,12 @@ public void add(Category newCategory) {
     /**
      * Replaces the category {@code target} in the list with {@code editedCategory}.
      * {@code target} must exist in the list.
-     * @param toRemove Category to remove
+     * @param category Category to remove
      */
-    public void remove(Category toRemove) {
-        requireNonNull(toRemove);
+    public void remove(Category category) {
+        requireNonNull(category);
 
-        if (!internalListOfCategories.remove(toRemove)) {
+        if (!internalListOfCategories.remove(category)) {
             //Throw an exception here later
```

Comment:
This part should be implemented by throwing an exception if the category to remove not being present in the list. If the Category was already checked to have been contained earlier, then this need not be an if statement.